### ISSUE TITLE
The sender and receiveer are not labeled in conversations for VoiceOver users on Signal for iOS version 2.36.0.7

### ISSUE THREAD

**Issue by santiagoblue7** [signalapp/Signal-iOS#4142]
When using Signal for iOS version 2.36.0.7 with the VoiceOver accessibility feature, the sender and receiveer in conversations are not labeled on the individual messages. 

Steps to reproduce: 

Step 1: Turn on VoiceOver by going to settings, general, accessibility, VoiceOver, and turning it on. 
Step 2: Using VoiceOver gestures, open the Signal app. 
Step 3: Double tap on a conversation, and flick through the individual messages. You will notice that VoiceOver will read the messages themselves, however, it won't read the sender ór receiveer's contact name.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by santiagoblue7** [comment]
I should also mention this bug is present on the stable release as well, which is 2.36.1. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by mrkiko** [comment]
I think this has been resolved. Can you confirm?

**Comment by santiagoblue7** [comment]
Hello, 
The latest beta build hasn't resolved this bug, however; I am aware that accessibility bug resolutions are on the works. 

Santiago
